### PR TITLE
Update bettertouchtool to 2.608

### PR DESCRIPTION
**Pull request by brianbrownton**
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` left no offenses.
- [x] The commit message includes the cask’s name and version.

### CODE REVIEW

**Comment by brianmorton** [comment]
I would love to get some feedback on the following:

The appcast currently lists latest as 2.606 - however the link on the appcast goes to a 404. Sniffing around the site I did find this:

https://bettertouchtool.net/releases/
image

This may be better as an appcast (or at least has the potential to be)... however care would need to be taken. I think the reason the 2.606 on the appcast link is broken is because on that releases page it is clear the filename is `btt2.606_alpha.zip` which is a little worrying as the releases  page didn't indicate it was an alpha...

In summary, I guess what I'm saying is that I think `https://bettertouchtool.net/releases/` would be a better appcast as long as we don't update to anything which says `alpha`, `beta`, or `rc` in the filename.

Thoughts?

**Comment by vitorgalvao** [comment]
Sounds good.

**Comment by brianmorton** [comment]
I have updated this PR with those changes.

I also changed the homepage, as the old homepage was just redirecting to what I have submitted.
